src/main/java/cs2103/v15_1j/jimjim/ui/BottomPaneController.java

```diff
@@ -48,8 +53,10 @@
 	private final double EXECUTE_BTN_HEIGHT = 30.0;
 	private final double HELP_BTN_WIDTH = 30.0;
 
-	public BottomPaneController(MainViewController con){
+	public BottomPaneController(MainViewController con, Stage primaryStage, String filePath){
 		this.con = con;
+		this.primaryStage = primaryStage;
+		this.filePath = filePath;
```

Comment:
better to just keep asking the controller than to keep a version of your own and then not synchronized